### PR TITLE
Estonian language

### PR DESCRIPTION
**Pull request by nondevspell**
I Translated two docs into estonian language add them please

My discord is spellecrin

### CODE REVIEW

File: svelte/src/libs/translations/languages/Estonian language [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
```
Comment (getneil) [inline svelte/src/libs/translations/languages/Estonian language:1]: @nondevspell thank you for contributing but the file type is wrong. it be a json file. ee.json

Comment (nondevspell) [reply svelte/src/libs/translations/languages/Estonian language:1]: Check it now please

Comment (getneil) [reply svelte/src/libs/translations/languages/Estonian language:1]: @nondevspell still wrong.

Comment (getneil) [reply svelte/src/libs/translations/languages/Estonian language:1]: @nondevspell please check as reference: https://github.com/pkgxdev/ossapp/pull/851

Comment (nondevspell) [reply svelte/src/libs/translations/languages/Estonian language]: maybe now?

Comment (getneil) [reply svelte/src/libs/translations/languages/Estonian language]: yeah but please fix the conflicts.

Comment (nondevspell) [reply svelte/src/libs/translations/languages/Estonian language]: try it know I think all is fixed

Comment (getneil) [reply svelte/src/libs/translations/languages/Estonian language]: the pr is still showing it has conflicts check below

Comment (nondevspell) [reply svelte/src/libs/translations/languages/Estonian language]: image
I solved conflicts but no changes

Comment (nondevspell) [reply svelte/src/libs/translations/languages/Estonian language]: can you try to edit my PR?

Comment (nondevspell) [reply svelte/src/libs/translations/languages/Estonian language]: Maybe I need to put ee.json before kr.json as in japanese reference

**Comment by nondevspell** [comment]
Nothing is changed...still conflicts

**Pull request closed without merging**